src/main/java/co/pragra/attendancesystem/controller/CohortController.java

```diff
@@ -0,0 +1,37 @@
+package co.pragra.attendancesystem.controller;
+
+import co.pragra.attendancesystem.entity.Cohort;
+import co.pragra.attendancesystem.service.CohortService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class CohortController {
+
+    @Autowired
+    private CohortService service;
+
```

Comment:
pls. add a Put mapping for updateCohort

Reply:
working on update method and test methods